gravitee-am-repository/gravitee-am-repository-jdbc-api/src/main/java/io/gravitee/am/repository/jdbc/provider/utils/TlsOptionsHelper.java

```diff
@@ -30,21 +30,25 @@
 public class TlsOptionsHelper {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(TlsOptionsHelper.class);
+    private static final String POSTGRESQL = "postgresql";
+    private static final String MYSQL = "mysql";
+    private static final String MARIADB = "mariadb";
+    private static final String SQLSERVER = "sqlserver";
 
     public static ConnectionFactoryOptions.Builder setSSLOptions(ConnectionFactoryOptions.Builder builder, Environment environment, String prefix, String driver) {
         final boolean useSSL = Boolean.valueOf(environment.getProperty(prefix + "sslEnabled", "false"));
         if (useSSL) {
             switch (driver) {
-                case "postgresql":
+                case POSTGRESQL:
                     builder = postgresOptions(builder, environment, prefix);
                     break;
-                case "mysql":
+                case MYSQL:
                     builder = mysqlOptions(builder, environment, prefix);
                     break;
-                case "mariadb":
+                case MARIADB:
                     builder = mariadbOptions(builder, environment, prefix);
                     break;
-                case "sqlserver":
+                case SQLSERVER:
                     builder = sqlServerOptions(builder, environment, prefix);
```

Comment:
**Suggestion**: in the same way we force the SSL to false for JDBC if the settings is not set in the gravitee.yaml, I think we should also force the SSL option to false for R2DBC. wdyt ?

Reply:
Done for R2DBC section.